src/scripts/main.js

```diff
@@ -1,3 +1,59 @@
 'use strict';
 
 // write code here
+
+const employees = document.querySelectorAll('li');
+const listEmployees = document.querySelector('ul');
```

Comment:
The variable `listEmployees` should be declared inside the `sortList` function, because it is not used anywhere else. In general, you should always try to limit the scope of your variables as much as possible to avoid potential conflicts and to make the code easier to understand.